core/src/test/java/org/elasticsearch/index/query/TemplateQueryBuilderTest.java

```diff
@@ -18,21 +18,41 @@
  */
 package org.elasticsearch.index.query;
 
+import org.apache.lucene.search.Query;
 import org.elasticsearch.common.xcontent.XContentBuilder;
 import org.elasticsearch.common.xcontent.XContentFactory;
 import org.elasticsearch.script.ScriptService.ScriptType;
 import org.elasticsearch.script.Template;
-import org.elasticsearch.test.ESTestCase;
 import org.junit.Test;
 
 import java.io.IOException;
 import java.util.HashMap;
 import java.util.Map;
 
-/**
- * Test building and serialising a template search request.
- * */
```

Comment:
Could be perceived as such, but here this is just to be consistent. If we start adding comments here, then we should do so for all the other classes.